development/Dockerfile

```diff
@@ -12,6 +12,8 @@ ENV PATH="${PATH}:/root/.local/bin" \
     INFRAHUB_FRONTEND_DIRECTORY="/opt/infrahub/frontend/app" \
     INFRAHUB_DOCS_DIRECTORY="/opt/infrahub/docs"
 
+RUN groupadd -r -g 1000 infrahub && useradd --no-log-init -u 1000 --home-dir /opt/infrahub -r -g infrahub infrahub
+
```

Comment:
I don't think the changes in `Dockerfile` are required.
Doing this is a breaking change for existing Infrahub instances regarding permissions (since the directories and files were created using `root`), I'd be careful with that... if you really need those changes.